src/bin/nydus-image/core/bootstrap.rs

```diff
@@ -560,7 +560,10 @@ impl Bootstrap {
             (0, 0)
         };
 
-        let orig_meta_addr = bootstrap_ctx.nodes[0].offset;
+        // Leave one block between the superblock and the inodes
+        // to avoid root inode use 0 as nid,
+        // which may cause the `..` disappear in some subdirs of root directory.
```

Comment:
The "WHY" is not really clear to me. Why would it cause `..` to disappear in some subdirs of the root directory?

Also when the comments says `Leave one block between the superblock and the inodes`, the meta address is moved one block backward. Wouldn't it cause bootstrap to overlap with the inode table?